packages/client/src/store/modules/grants.js

```diff
@@ -94,6 +94,16 @@ export default {
       await fetchApi.deleteRequest(`/api/organizations/:organizationId/keywords/${keywordId}`);
       dispatch('fetchKeywords');
     },
+    exportCSV(context, queryParams) {
+      const query = Object.entries(queryParams)
+        // filter out undefined and nulls since api expects parameters not present as undefined
+        // eslint-disable-next-line no-unused-vars
+        .filter(([key, value]) => value || typeof value === 'number')
+        .map(([key, value]) => `${encodeURIComponent(key)}=${encodeURIComponent(value)}`)
+        .join('&');
+      const navUrl = fetchApi.addOrganizationId(`/api/organizations/:organizationId/grants/exportCSV?${query}`);
+      window.location = navUrl;
```

Comment:
this will open a new tab and close it or will the user need to click back to come back to the site?

Reply:
I'll test in FF and Safari too, but at least in Chrome if you click a link or navigate in the same window like this to a URL that serves `Content-Disposition: attachment`, it just downloads the file without actually taking the user away from the page they're on. I suppose it's possible they'd navigate away to an error page if the export endpoint errors somehow, but I don't think clicking back is a big deal in that scenario.

Reply:
Tested and confirmed FF and Safari on Mac have similar behavior.